### ISSUE TITLE
[Issue][1.0.0] Typescript errors

### ISSUE THREAD

**Issue by alexisvigoureux** [deligenius/aws-sdk-v3-nest#9]
Hello, I'm trying to migrate my application from version 0.3.0 to version 1 and I'm getting the following Typescript errors:

```
error TS2322: Type 'typeof S3Client' is not assignable to type 'ClassDefinition'.
Types of construct signatures are incompatible.
    Type 'new (...[configuration]: [] | [S3ClientConfig]) => S3Client' is not assignable to type 'new (...args: unknown[]) => unknown'.
      Types of parameters '__0' and 'args' are incompatible.
        Type 'unknown[]' is not assignable to type '[] | [S3ClientConfig]'.
          Type 'unknown[]' is not assignable to type '[S3ClientConfig]'.
            Target requires 1 element(s) but source may have fewer.
```

```
error TS2322: Type '(configService: ConfigType<typeof applicationConfig>) => Promise<S3Client>' is not assignable to type '(...args: unknown[]) => unknown'.
  Types of parameters 'configService' and 'args' are incompatible.
    Type 'unknown' is not assignable to type '{ executionContext: string; nodeEnv: string; encryption: { sharedSecretKeyBase: string; }; ... 12 more ...; geolocation: { ...; }; }'.
```

Here is the code 

```
import { ListBucketsCommand, S3Client } from '@aws-sdk/client-s3'
import { AwsSdkModule } from 'aws-sdk-v3-nest'
...
    AwsSdkModule.registerAsync({
      imports: [ConfigModule.forFeature(applicationConfig)],
      inject: [applicationConfig.KEY],
      clientType: S3Client,
      isGlobal: true,
      useFactory: async (configService: ConfigType<typeof applicationConfig>) => {
       .........
      },
    }),
```

Was it intented to migrate all the `any` type to `unknown` in [v1](https://github.com/deligenius/aws-sdk-v3-nest/commit/fe181772e5b33f34dbb7d37dd54c8ff0635cf980#diff-9627df1c03612af632fd36ada00925affe222cd19c88869fb82647e29b753054)?

**Comment by gjuoun** [comment]
sorry I didn't cover this case, I'll revert it to any

**Comment by gjuoun** [comment]
It's resolved, please use version 1.0.1
